controllers/v1/observationsController.js

```diff
@@ -1185,6 +1186,8 @@ module.exports = class Observations extends Abstract {
                         newSolutionDocument.parentSolutionId = newBaseSolution._id;
                         newSolutionDocument.isReusable = false;
                         newSolutionDocument.externalId = frameworkDocument.externalId;
+                        let hashedLink = md5(newSolutionDocument._id+"###"+req.userDetails.userId);
```

Comment:
@priyanka-TL - A link has to be sent in the response of this API and generate the link after creating solution below. If the app name is not given, give default app name as samiksha.